app/controllers/api/stages_controller.rb

```diff
@@ -36,4 +50,17 @@ def deploy_group_ids
   def deploy_groups
```

Comment:
params should not be deploy_group_ids, but params[:stage][:deploy_group_ids] -> open for future additions ?